pyproject.toml

```diff
@@ -31,6 +39,13 @@ dev = [
     "pytest-cov",
     "types-PyYAML",
 ]
+docs = [
+    "sphinx",
+    "sphinx-rtd-theme",
+    "bluepy",
```

Comment:
remove bluepy dependency; not required and causes issues with local install

Reply:
fixed